nano/node/network.cpp

```diff
@@ -468,7 +468,7 @@ class network_message_visitor : public nano::message_visitor
 
 void nano::network::process_message (nano::message const & message, std::shared_ptr<nano::transport::channel> const & channel)
 {
-	node.stats.inc (nano::stat::type::message, nano::to_stat_detail (message.header.type), nano::stat::dir::in);
```

Comment:
We generally use fully qualified names when invoking functions, in this case why are we making the call less obvious? Why to_stat_detail and not nano::to_stat_detail?

Reply:
Using ADL is a shorter, more bulletproof and generally better ways to write `to_stat_detail`, `to_log_detail` and `to_string` functions. Why do you want to keep fully qualified namespace here?

Reply:
I understood we had a rule that we use fully qualified everywhere.

Reply:
Anyway just asking questions, I'd like to clarify what the rules are.